website/src/_data/customFilters.js

```diff
@@ -0,0 +1,14 @@
+var nunjucks = require('nunjucks');
```

Comment:
No "var"! 90% of the time it should be "const" ([see Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let))